docs/includes/excel-1_15.md

```diff
@@ -1,9 +1,136 @@
 | Class | Fields | Description |
 |:---|:---|:---|
+|[Application](/javascript/api/excel/excel.application)|||
```

Comment:
None of these new APIs are actually in ExcelApi 1.15. @AlexJerabek could PR #1274 have triggered unexpected changes this file?